diff -urN dame-0.27.orig/dame.src/dame.desktop dame-0.27/dame.src/dame.desktop
--- dame-0.27.orig/dame.src/dame.desktop	2004-11-30 18:01:44.393506296 +0100
+++ dame-0.27/dame.src/dame.desktop	2004-11-30 18:05:16.594246880 +0100
@@ -1,13 +1,15 @@
 [Desktop Entry]
 Name=Dame
-Comment=Damebrett
+Comment=Checkerboard
+Comment[cz]=Pl√°n d√°ma
 Comment[de]=Damebrett
-Comment[en]=checkerboard
-Comment[fr]=damier
-Comment[it]=gioco di dama
-Comment[cz]=pl·n d·ma
-Comment[fi]=tammi peli
+Comment[fi]=Tammi peli
+Comment[fr]=Damier
+Comment[it]=Gioco di dama
+Comment[pl]=Warcaby
 Exec=dame
 Icon=dame.png
-Terminal=0
+Terminal=false
 Type=Application
+Categories=GTK;GNOME;Game;BoardGame;
+Encoding=UTF-8
